installer/deb/scripts/postinst

```diff
@@ -59,6 +59,10 @@ set_local_dns() {
 
 # Download and install mkcert, generate a certificate for MassaStation.
 generate_certificate() {
+    while ! ifconfig | grep "192.168." > /dev/null; do
+        echo "waiting for network manager to be up and running"
+        sleep 2
```

Comment:
WARN: this is a potential infinite loop

Reply:
chat gpt suggestion:

```
#!/bin/bash

counter=0
timeout=10  # Timeout in seconds

while ! ifconfig | grep "192.168." > /dev/null && ((counter < timeout)); do
    echo "Waiting for network manager to be up and running"
    sleep 1
    ((counter+=1))
done

if ((counter >= timeout)); then
    echo "Timeout reached. Network manager is not available."
else
    echo "Network manager is up and running."
fi
```

Reply:
I think you should create a `wait_for_network` function that contains what ChatGPT gave you, and call it at the begining of `generate_certificate`

Reply:
yep!